hack/manual-testing.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
```

Comment:
@jsturtevant please review. I've removed the docker push step since it's not required anymore with this change and added the required environment variables so that the script is the only thing that needs to be run to deploy a test cluster.